manim/config.py

```diff
@@ -1,326 +1,427 @@
-import argparse
-import colour
+"""
+config.py
+---------
+
+Process the manim.cfg file and the command line arguments into a single
+config object.
+
+"""
+
 import os
 import sys
-import types
-
-from .utils.tex import *
-from . import constants
-from . import dirs
+import argparse
+import configparser
+import colour
+from .utils.tex import TexTemplateFromFile, TexTemplate
 from .logger import logger
+from . import constants
+
+__all__ = ["file_writer_config", "config", "camera_config"]
+
+
+def _parse_config(config_parser, args):
```

Comment:
Since this is private it should be ok to not be specific, but, if there are new, you could consider documenting those private functions in order to help future programmers understand the function i think